ovos_plugin_common_play/ocp/search.py

```diff
@@ -40,13 +50,14 @@ def reset(self):
         self.query_timeouts = self.settings.min_timeout
 
     @property
-    def settings(self):
+    def settings(self) -> dict:
         if self.ocp_search:
             return self.ocp_search.settings
-        return OCPSettings()
+        # TODO: Default Settings?
```

Comment:
since self.settings now respects the XDG spec they can be loaded from there

but only if parent object is not available since it may include updated values from mycroft.conf

Reply:
previous behavior would just return defaults, should this just return an empty dict and use defaults, or check for a config file?

Reply:
now that all settings are jsonstorage it should return any previous values, its like reading from a config file

Reply:
should we explicitly populate the settings file with default values on plugin load if it doesnt exist?